webapp/src/components/musicEditor/MusicEditor.tsx

```diff
@@ -367,27 +377,63 @@ export const MusicEditor = (props: MusicEditorProps) => {
 
         // First, check if we are erasing
         if (eraserActive) {
-            for (let i = 0; i < currentSong.tracks.length; i++) {
-                if (hideTracksActive && i !== selectedTrack) continue;
-
-                const track = currentSong.tracks[i];
-                const instrument = track.instrument;
-                const existingEvent = findPreviousNoteEvent(currentSong, i, end.exactTick);
-
-                const isAtCoord = (note: pxt.assets.music.Note) => {
-                    const isBassClef = isBassClefNote(instrument.octave, note, isDrumTrack);
-
-                    if (isBassClef !== end.isBassClef) return false;
-
-                    return noteToRow(instrument.octave, note, isDrumTrack) === end.row;
-                }
+            if (end.row > 11 || end.row < 0) return;
```

Comment:
Not sure I'm quite following what the 11 & 12 magic numbers are coming from. Could these be constants with meaningful names?